### PR TITLE
chore: introduce corba as the cli scafford

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance

- Related issues

https://github.com/apache/apisix-dashboard/issues/739

### CODE REVIEW

**Comment by membphis** [comment]
@tokers any news?

**Comment by tokers** [comment]
@membphis Soory i forgot this, i will update it as soon as possible.

**Comment by tokers** [comment]
@nic-chen @membphis Now this PR can be reviewed.

**Comment by nic-chen** [comment]
@tokers conflict again..

**Comment by tokers** [comment]
> @tokers conflict again..

Resolved.

**Comment by nic-chen** [comment]
@tokers 
CI failed and we need test case for the change.

**Comment by tokers** [comment]
> @tokers
> CI failed and we need test case for the change.

OK, i will resolve it as soon as possible.

**Review by nic-chen — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
@tokers need to resolve the conflicts

File: api/go.sum [hunk 1]
```diff
@@ -1,30 +1,86 @@
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (membphis) [inline api/go.sum:1]: Need to confirm the license of these new dependencies

**Review by membphis — CHANGES_REQUESTED** [review]

**Comment by tokers** [comment]
License about new dependencies:

* github.com/cameront/go-jsonpatch MIT
* github.com/coreos/bbolt MIT
* github.com/dustin/go-humanize MIT
* github.com/golang/groupcache Apache 2.0
* github.com/grpc-ecosystem/go-grpc-middleware Apache 2.0
* github.com/grpc-ecosystem/grpc-gateway BSD 3.0
* github.com/jonboulle/clockwork Apache 2.0
* github.com/prometheus/client_golang Apache 2.0
* github.com/spf13/cobra Apache 2.0
* github.com/tmc/grpc-websocket-proxy MIT
* golang.org/x/time BSD 3.0
* sigs.k8s.io/yaml MIT

**Comment by juzhiyuan** [comment]
ping @membphis

**Review by gxthrj — APPROVED** [review]

File: api/go.sum (hunk 1 above)
Comment (tokers) [reply api/go.sum:1]: License of all packages are checked, everything is OK.

- github.com/cameront/go-jsonpatch MIT
- github.com/coreos/bbolt MIT
- github.com/dustin/go-humanize MIT
- github.com/golang/groupcache Apache 2.0
- github.com/grpc-ecosystem/go-grpc-middleware Apache 2.0
- github.com/grpc-ecosystem/grpc-gateway BSD 3.0
- github.com/jonboulle/clockwork Apache 2.0
- github.com/prometheus/client_golang Apache 2.0
- github.com/spf13/cobra Apache 2.0
- github.com/tmc/grpc-websocket-proxy MIT
- golang.org/x/time BSD 3.0
- sigs.k8s.io/yaml MIT

Comment (membphis) [reply api/go.sum:1]: nice

**Review by membphis — APPROVED** [review]

**Comment by starsz** [comment]
@tokers , What happen to the `.actions/openwhisk-utilities`?
Please see the file changes.

**Comment by tokers** [comment]
> @tokers , What happen to the `.actions/openwhisk-utilities`?
> Please see the file changes.

We don't reference it as a submodule, just download it and use it in CI.

**Pull request merged**